frontend/packages/kubevirt-plugin/src/components/vms/vm-resource.tsx

```diff
@@ -214,20 +214,12 @@ export const VMDetailsList: React.FC<VMResourceListProps> = ({
         {launcherPod && ipAddrs}
       </VMDetailsItem>
 
-      <VMDetailsItem
-        title="Hostname"
-        idValue={prefixedID(id, 'hostname')}
-        isNotAvail={!getGuestAgentInfoHostname(guestAgentInfo)}
-      >
-        {getGuestAgentInfoHostname(guestAgentInfo)}
+      <VMDetailsItem title="Hostname" idValue={prefixedID(id, 'hostname')} isNotAvail={!hostname}>
```

Comment:
This may change based on @matthewcarleton's input on https://github.com/openshift/console/pull/5741#discussion_r440735222, but we'll go with this for now.

Reply:
:+1: